### PR TITLE
emacs

### LINKED ISSUE #182: emacs

### PR DESCRIPTION
**Pull request by TianjinDora**
# Advanced HW 9 submission

Closes #182

Name: Shuo Chen
Uniqname: shuochen

### CODE REVIEW

**Comment by mterwill** [comment]
You committed a temp file here, please remove it!

**Comment by TianjinDora** [comment]
OK, removed.

Shuo

On Fri, Mar 25, 2016 at 11:13 PM, Matt Terwilliger <notifications@github.com

**Comment by ppannuto** [comment]
@DavidSnider As our resident emacs expert, are there any other things that are useful to be included here? I have a few comments, but I'll leave final merge to you.

**Comment by ppannuto** [comment]
Can you move this from /tools to /editors ?

**Comment by ppannuto** [comment]
You have some formatting issues:

screen shot 2016-03-26 at 14 18 15

Be sure to check out how this looks locally while you make changes.

File: _commands/tools/emacs.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+~~~
+
+##### Break it down
+
+ * Set the Emacs window’s font to that specified by font. You will find the various X fonts
+   in the /usr/lib/X11/fonts directory. Note that Emacs will only accept fixed width fonts. 
+   Under the X11 Release 4 font-naming conventions, any font with the value "m" or "c" in the 
+   eleventh field of the font name is a fixed width font. Furthermore, fonts whose name are of
+   the form widthxheight are generally fixed width, as is the font fixed. See **xlsfonts**(1) for 
+   more information.
```
Comment (ppannuto) [inline _commands/tools/emacs.md]: This looks copied from a man page. The goal isn't to copy man documentation. How much of this paragraph is useful for the average c4cs student? The idea is to pull out the things that are actually useful, in this case something closer to "The font must be installed on your system to work. Some fonts installed that I like are [...]. For all the details on fonts, check out the `xlsfonts` manpage"

File: _commands/tools/emacs.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+~~~ bash
+$ emacs -geometry 70 filename
+~~~
+
+##### Break it down
+
+ * Set the Emacs window’s width, height, and position as specified. The geometry specification
+   is in the standard X format; see X(1) for more information. The width and height are specified 
+   in characters; the default is 80 by 24.
```
Comment (ppannuto) [inline _commands/tools/emacs.md]: Same comment here. People shouldn't have to look up information in another man page for this quick reference to be useful to them.

**Comment by DavidSnider** [comment]
A few nits:

you say

> emacs is used to open a file in emacs
> 
> $ emacs filename


While mostly true, I have been known to just open emacs like `emacs -nw` if I want to take quick notes.

In your description of `emacs -q` you mention an init file. While I know what an init file is, the casual reader may not.

> Open Emacs and goto line 7.

I think you probably mean "move to cursor to line 7".

> $ emacs -font 12 filename

filename still unnecessary. Ypu need to fix that in a few places.

Other than that, You probably should mention how it reads a .emacs .

**Comment by themoh** [comment]
We'll be closing all pull requests at midnight tonight, so please have this done by then.

File: _commands/editors/emacs.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 emacs
 --
 
-`emacs` is used to open a file in emacs
+`emacs` is edit files in emacs
```
Comment (DavidSnider) [inline _commands/editors/emacs.md]: Please fix this typo.

**Pull request merged**